src/managing-workflow/platform-logging.md

```diff
@@ -11,20 +11,14 @@ If the message is from the [Workflow Router](https://github.com/deis/router) we
 Logger then acts as a consumer reading messages off of the NSQ logs topic storing those messages in a local Redis instance. When a user wants to retrieve log entries using the `deis logs` command we make an HTTP request from Controller to Logger which then fetches the appropriate data from Redis.
 
 ## Configuring Off Cluster Redis
-Even though we provide a redis instance with the default Workflow install. It is recommended that operators use a 3rd party source like Elasticache or similar offering. This way your data is durable across upgrades or outages. If you have a 3rd party Redis installation you would like to use all you need to do is set the following values in `generate_params.toml` within your chart's tpl directory.
+
+Even though we provide a redis instance with the default Workflow install, it is recommended that operators use a third-party source like Elasticache or similar offering. This way your data is durable across upgrades or outages. If you have a third-party Redis installation you would like to use all you need to do is set the following values in your chart:
```

Comment:
Should we mention how the `logger redis` values can be set via `helm`?  (Something similar to [configuring object storage](https://github.com/deis/workflow/blob/687e916d0b955e467e6cb42263fe230f45560a39/src/installing-workflow/configuring-object-storage.md#step-4-configure-workflow-chart) perhaps?)